open-apparel-registry/open-apparel-registry
# Expand ExtendedField.FIELD_CHOICES

## Overview

Creates a data migration to add additional ExtendedField.FIELD_CHOICES,
specifically facility_type, processing_type, product_type, and
parent_company.

Connects #1581 

## Demo

<img width="858" alt="Screen Shot 2022-01-19 at 12 09 55 PM" src="https://user-images.githubusercontent.com/21046714/150202010-de84f401-3e18-4f7c-8ec1-e1f6049060b9.png">

## Notes

I temporarily rebased this on the expanded [facility details PR](https://github.com/open-apparel-registry/open-apparel-registry/pull/1593), and was able to manually add extended fields of the new types and view them in the facility details.

## Testing Instructions

* Run `./scripts/manage migrate`
* Run `./scripts/server`
* Navigate to [the extended fields admin](http://localhost:8081/admin/api/extendedfield/) and update or create an ExtendedField. 
* The list of field types should include facility_type, processing_type, product_type, and parent_company.

## Checklist
